cloud/network-billing.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Network Billing
+description: This document outlines how network usage is charged in RisingWave Cloud.
+sidebarTitle: Network Billing
+---
+
+## Pricing Dimensions
+
+Network usage is billed based on the following dimensions:
+
+### 1. Public Network Traffic
+
+Network traffic charges are based on the volume of data moved across public internet, measured in gigabytes (GB). This includes:
+
+- **Ingress** (Data entering RisingWave):
+    - **Streaming Data Integration**: Continuously ingesting real-time data streams from external message brokers or sources into RisingWave for immediate processing.
+    - **Batch Data Imports**: Loading data from external systems, such as databases or cloud storage, into RisingWave for analytics or processing.
+- **Egress** (Data leaving RisingWave):
+    - **Data Export to External Destinations**: Sending processed data to external databases, message brokers, or other sinks.
+    - **Client Application Queries**: When client applications retrieve query results from RisingWave, the outbound data transfer constitutes egress traffic.
+
+### 2. PrivateLink Usage
+
+PrivateLink enables secure, private connections between your Virtual Private Cloud (VPC) and RisingWave Cloud without exposing traffic to the public internet. Charges for PrivateLink are based on:
+
+- **PrivateLink Hours**: Billed per hour for each active PrivateLink endpoint.
+- **PrivateLink Throughput**: Billed per GB of data transferred through the PrivateLink connection.
+
+For more details about how PrivateLink works in RisingWave Cloud, refer to the [PrivateLink Overview](https://docs.risingwave.com/cloud/privatelink-overview).
+
+## Pricing Calculation
+
+Network usage costs are calculated based on actual consumption. Here's a detailed breakdown by cloud provider:
+
+### AWS Pricing
+
+| Category | US and EU Regions | Singapore |
+| --- | --- | --- |
+| **Ingress** | $0.07 /GB | $0.082 /GB |
+| **Egress** | $0.17 /GB | $0.215 /GB |
+| **PrivateLink Ingress Throughput** | $0.013 /GB | $0.013 /GB |
+| **PrivateLink Egress Throughput** | $0.013 /GB | $0.013 /GB |
+| **PrivateLink Endpoint Hours** | $0.03 /hr | $0.03 /hr |
```

Comment:
Prefer no blanks before and after slash, like `$0.07/GB`. 

https://en.wikipedia.org/wiki/Slash_%28punctuation%29#Spacing